pkgs/misc/drivers/hplip/default.nix

```diff
@@ -219,6 +222,10 @@ python3Packages.buildPythonApplication {
       --replace {,${utillinux}/bin/}logger \
       --replace {/usr,$out}/bin
     remove-references-to -t ${stdenv.cc.cc} $(readlink -f $out/lib/*.so)
+  '' + stdenv.lib.optionalString withQt5 ''
+    for f in $out/bin/hp-*;do
+      wrapQtApp $f
```

Comment:
FYI, I'm pretty sure the binaries are wrapped twice now. It should have been:

```
  preFixup = ''
    makeWrapperArgs+=("''${qtWrapperArgs[@]}")
  ''
```

But it's not that bad...

Reply:
Yeah, if someone wants to clean this up, I'd happily review a PR, but don't feel it a pressing enough issue to do it myself :-)